scripts/main.js

```diff
@@ -0,0 +1,94 @@
+const myLibrary = [];
```

Comment:
## Modularise this file
- [ ] Do you mind keeping `application` logic separated from `DOM manipulation` tasks as directed in this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/dom_logic_separation_requirement.png) from Pathwright?  
 
I recommend this because `separation of concerns` as it is known is the idea that each module or layer in an application should only be responsible for one thing and should not contain code that deals with other things. Separating concerns reduces code complexity by breaking a large application down into many smaller units of encapsulated functionality. Please feel free to read more about this suggestion [here](https://www.oreilly.com/library/view/programming-javascript-applications/9781491950289/ch05.html).